src/components/Dropdown/MultiSelectDropdown.jsx

```diff
@@ -86,15 +86,13 @@ export class MultiSelectDropdown extends Component {
 
   render() {
     const { value, placeholder, textField, ...rest } = this.props;
-
-    const { data, isOpen, inputValue } = this.state;
-
+    const { data, isOpen, inputValue = '' } = this.state;
     return (
       <Downshift
         {...rest}
         onChange={this.onValueChange}
         isOpen={isOpen}
-        inputValue={inputValue}
+        inputValue={`${inputValue}`}
```

Comment:
Hvorfor wrappe det i literals?

Reply:
det var for å stoppe klagingen. react likte vist ikke at noe i downshift endret verdien. Men når inputen er bare "tekst" og ikke en variabel så klager den ikke noe mer.

Reply:
men inputvalue er jo bare tekst?